integration/nwo/fabric/network/checks.go

```diff
@@ -40,7 +40,8 @@ func (n *Network) CheckTopologyOrderers() {
 			ports[portName] = n.Context.ReservePort()
 		}
 		n.Context.SetPortsByOrdererID(n.Prefix, o.ID(), ports)
-		n.Context.SetHostByOrdererID(n.Prefix, o.ID(), "0.0.0.0")
+		// TODO: allow to set the orderer endpoint via topo
+		n.Context.SetHostByOrdererID(n.Prefix, o.ID(), "127.0.0.1")
```

Comment:
we don't even need this anymore

Reply:
That is, I believe that all components/services started by NWO should by default run on `127.0.0.1` and we can allow explicitly via topologo.go to change to different listing addresses if needed.